plenum/test/signing/test_signing_out_of_ledger.py

```diff
@@ -0,0 +1,87 @@
+import json
+import types
+
+import pytest
+from indy.did import create_and_store_my_did
+from indy.ledger import build_nym_request
+
+from plenum.common.constants import NYM, STEWARD, ROLE, VERKEY
+from plenum.common.exceptions import UnauthorizedClientRequest, RequestNackedException
+from plenum.common.txn_util import get_request_data
+from plenum.common.util import randomString
+from plenum.server.request_handlers.utils import get_nym_details
+from plenum.test.helper import sdk_get_and_check_replies
+from plenum.test.pool_transactions.helper import sdk_sign_and_send_prepared_request
+
+NEW_ROLE = None
+
```

Comment:
I would rename the test file to `test_create_did_without_endorser`

Reply:
ok